invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SearchResultsBulkActionsManager.js

```diff
@@ -0,0 +1,82 @@
+import { BulkActionsContext } from "./context";
+import React, { Component } from "react";
+import Overridable from "react-overridable";
+import _hasIn from "lodash/hasIn";
+
+class SearchResultsBulkActionsManager extends Component {
+  constructor(props) {
+    super(props);
+
+    this.selected = {};
+    this.state = { allSelected: false, selectedCount: 0 };
+  }
+
+  addToSelected = (rowId) => {
+    const { selectedCount } = this.state;
+
+    if (_hasIn(this.selected, `${rowId}`)) {
```

Comment:
is this if for a `setdefault` implementation?
```suggestion
  this.selected[rowId] = this.selected[rowId] === undefined ? true : !this.selected[rowId]
```
or
```suggestion
  this.selected[rowId] = rowId in this.selected ? !this.selected[rowId] : true;
```